endpoints/bookstore-grpc/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM gcr.io/google_appengine/python
```

Comment:
do you mean adding a comment in this Dockerfile? something like this:
# https://github.com/GoogleCloudPlatform/python-runtime

Or adding this link in the README.md?